src/inflation/db/datomic/db.clj

```diff
@@ -0,0 +1,41 @@
+(ns inflation.db.datomic.db
+  (:require [datomic.api :as d]))
+
+(def db-uri "datomic:dev://localhost:4334/inflation")
```

Comment:
Unless you do want to create a transactor to test how it is done, you can use datomic in memory (e.g. "datomic:mem://inflation").